### PR TITLE
add evm examples

### PR DESCRIPTION
**Pull request by ntduan**

### CODE REVIEW

**Comment by shunjizhan** [comment]
seems that CI is failing @ntduan

**Comment by ntduan** [comment]
> seems that CI is failing @ntduan

https://github.com/AcalaNetwork/bodhi.js/pull/113 Yes, I have submitted a new one

**Comment by shunjizhan** [comment]
:+1:

File: examples/dex/contracts/RecurringPayment.sol [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+contract RecurringPayment is ADDRESS {
+    uint period;
+    uint remainingCount;
+    uint amount;
+    address payable to;
+    bool private initialized;
+    ISchedule scheduler = ISchedule(ADDRESS.Schedule);
+
+    constructor(uint _period, uint _count, uint _amount, address payable _to) public {
```
Comment (gluneau) [inline examples/dex/contracts/RecurringPayment.sol:15]: The command `rush build` gives a warning here: 

```
Warning: Visibility for constructor is ignored. If you want the contract to be non-deployable, making it "abstract" is sufficient.
  --> contracts/RecurringPayment.sol:15:5:
   |
15 |     constructor(uint _period, uint _count, uint _amount, address payable _to) public {
   |     ^ (Relevant source part starts here and spans across multiple lines).

```

Comment (shunjizhan) [reply examples/dex/contracts/RecurringPayment.sol:15]: yeah I also see that warning, not sure if it will affect the functionality...

Comment (ntduan) [reply examples/dex/contracts/RecurringPayment.sol:15]: I have tested them and now all the example tests are OK

**Pull request merged**